buildozer/targets/ios.py

```diff
@@ -115,15 +115,30 @@ def toolchain(self, cmd, **kwargs):
         kwargs.setdefault('cwd', self.ios_dir)
         return self.buildozer.cmd(self._toolchain_cmd + cmd, **kwargs)
 
-    def xcodebuild(self, cmd='', **kwargs):
-        return self.buildozer.cmd(self._xcodebuild_cmd + cmd, **kwargs)
+    def xcodebuild(self, *args, **kwargs):
+        return self.buildozer.cmd(self._xcodebuild_cmd + ' '.join(arg for arg in args if arg is not None), **kwargs)
```

Comment:
This approach allows easily omit unspecified (None) arguments, which makes later invocation cleaner.